src/views/blocks/rsvp/form/quantity-input.php

```diff
@@ -35,9 +40,6 @@ class="tribe-tickets-quantity"
 	min="1"
 	value="1"
 	required
-	data-remaining="<?php echo esc_attr( $available ); ?>"
-	<?php if ( -1 !== $available ) : ?>
-		max="<?php echo esc_attr( $available ); ?>"
-	<?php endif; ?>
+	max="<?php echo esc_attr( $max_at_a_time ); ?>"
```

Comment:
Should we add `data-remaining` back with a separate available call?

Reply:
@sc0ttkclark confirmed via Slack DMs that the data attribute is okay to be removed, since the template's previous `@since` tag mentioned its removal as of `[4.11.1] 2019-12-19` (just that it wasn't _actually_ removed).